test/docker.sh

```diff
@@ -3,7 +3,6 @@
 #set -e
 
 test_node_versions="0.8.28 0.10.40 0.12.7 4.3.0 5.6.0"
```

Comment:
Should we be updating these (perhaps in a different PR) to in support versions of Node.js?

Reply:
Yep, good idea. I could do that as a separate commit or a separate PR.

Reply:
> 
> 
> Yep, good idea. I could do that as a separate commit or a separate PR.

Your choice 😄